commit 834a4ff4f2752631252cff4e3b2ba87bee62965b
Author: Valentina Mukhamedzhanova <vmukhame@redhat.com>
Date:   Thu Dec 10 16:24:18 2015 +0100

    yum-plugin-priorities: get all obsoletes, not just the newest. BZ#1269414

diff --git a/plugins/priorities/priorities.py b/plugins/priorities/priorities.py
index ce98583..006c545 100644
--- a/plugins/priorities/priorities.py
+++ b/plugins/priorities/priorities.py
@@ -111,7 +111,7 @@ def exclude_hook(conduit):
     if check_obsoletes and not conduit._base.conf.obsoletes:
         check_obsoletes = False
     if check_obsoletes:
-        obsoletes = conduit._base.up.rawobsoletes
+        obsoletes = conduit._base.pkgSack.returnObsoletes()
 
     # Build a dictionary with package priorities. Either with arch or
     # archless, based on the user's settings.
